exec-service-wasmer/src/wasmer_instance.rs

```diff
@@ -88,6 +82,27 @@ fn extract_wasmer_memory_name(wasmer_instance: &wasmer::Instance) -> Result<Stri
     }
 }
 
+fn push_middlewares(
+    compiler: &mut Singlepass,
+    compilation_options: &CompilationOptions,
+    executor_data: Rc<WasmerExecutorData>,
```

Comment:
Question for a later time, not related to this PR: should `WasmerExecutorData` be an `Arc` as well? 

E.g. the `metering` middleware instance below is an `Arc` using `Rc<WasmerExecutorData>`. Not sure if this is an issue or not.

Reply:
It should stay `Rc<WasmerExecutorData>`, it is not passed to any middleware, it is only used inside `fn push_middlewares` for printing. Only `opcode_cost` is passed and this is already an `Arc`.